samples/hello-oboe/src/main/cpp/PlayAudioEngine.cpp

```diff
@@ -223,7 +206,6 @@ void PlayAudioEngine::onErrorAfterClose(oboe::AudioStream *oboeStream, oboe::Res
 
 void PlayAudioEngine::restartStream(oboe::AudioStreamBuilder *builder) {
     LOGI("Restarting stream");
-    closeOutputStream();
     createPlaybackStream(builder);
```

Comment:
We still want to delete the old stream before we create the new stream because the new stream may need exclusive resources held by the old stream.

Reply:
This is taken care of in the `AudioStreamBuilder::openManagedStream` method.